app/pages/resource/ResourcePage.js

```diff
@@ -10,6 +10,7 @@ import { bindActionCreators } from 'redux';
 import Col from 'react-bootstrap/lib/Col';
 import Panel from 'react-bootstrap/lib/Panel';
 import Lightbox from 'lightbox-react';
+import axios from 'axios';
```

Comment:
You don't need axios for only this endpoint, I suggest to follow what already has been done for other endpoints